docs/dev/resources/networking.md

```diff
@@ -0,0 +1,32 @@
+# Networking Common Tasks and Information
+
+## Quick Access
+[Allow Connection to Sites on the Internet from a ProB workload](#allowing-connection-to-sites-on-the-internet)
+
+[Allow Connection to internal application](#allow-connection-to-internal-application)
+
+[DNS pitfalls](#dns-pitfalls-to-internal-application-connecting)
+
+### Allowing Connection to sites on the Internet
+This is a bit more niche, as our unclassified notebooks already have connection to the internet, but this can come up when we have a Protected B certified application like seen in https://github.com/StatCan/aaw-private/issues/125 where it's accessible to the internet, but unaccecssible from our Protected B workloads.
+
+To handle this we want to declare a [firewall_policy_rule_collection_group](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/firewall_policy_rule_collection_group) in our terraform files. The source address is the address associated with protected-b workloads. In the issue above we created it in our specific environment files as the `firewall.tf` file in our `terraform-azure-statcan-aaw-network` is mirrored and publically accessible.

```

Comment:
`The source address is the address associated with protected-b workloads.`

That still isn't clear, maybe something like:
`The source address is the subnet address of our protected-b node pool.`